kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/java/org/kie/server/services/jbpm/JbpmKieServerExtension.java

```diff
@@ -661,6 +661,11 @@ protected boolean isExecutorOnClasspath() {
         persistenceProperties.put("hibernate.transaction.jta.platform", config.getConfigItemValue(KieServerConstants.CFG_PERSISTANCE_TM, "org.hibernate.service.jta.platform.internal.JBossAppServerJtaPlatform"));
         persistenceProperties.put("javax.persistence.jtaDataSource", config.getConfigItemValue(KieServerConstants.CFG_PERSISTANCE_DS, "java:jboss/datasources/ExampleDS"));
 
+        System.getProperties().stringPropertyNames()
+                .stream()
+                .filter(name -> name.startsWith("hibernate.") || name.startsWith("javax.persistence"))
```

Comment:
I think here you can use a method based filter, for example:

.filter(JbpmKieServerExtension::isValidPersistenceKey)

so it can be reused other places if needed